examples/node_pool/main.tf

```diff
@@ -150,4 +150,9 @@ module "gke" {
       "net.core.netdev_max_backlog" = "20000"
     }
   }
+
+  node_pools_cgroup_mode = {
+    all     = "CGROUP_MODE_UNSPECIFIED"
+    pool-01 = "CGROUP_MODE_V2"
```

Comment:
Thanks @DrFaust92!

Looks like both `CGROUP_MODE_UNSPECIFIED` and `CGROUP_MODE_V2` result in `EFFECTIVE_CGROUP_MODE_V2`.  Let's set `all` or a different pool to `CGROUP_MODE_V1`, and update the relevant test data to `EFFECTIVE_CGROUP_MODE_V1` https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/master/test/integration/node_pool/testdata/TestNodePool.json so we can be certain this is working as intended.

Reply:
Done, lmk if worked. im having some trouble running these E2E so im testing in a hacky way

Reply:
Thanks @DrFaust92! Looks like all the nodepools are now testing as v1, except for pool-01 which is correctly v2.